src/generic/modal-dropzone/useModalDropzone.jsx

```diff
@@ -101,7 +101,7 @@ const useModalDropzone = ({
 
         setTimeout(() => {
           onClose();
-        }, 1000);
+        }, 0);
```

Comment:
is it needed to have timeout here? Maybe just `onClose();`?

Reply:
you are right, there is no need in timeout anymore!